modules/pytket-braket/docs/changelog.rst

```diff
@@ -5,6 +5,8 @@ Changelog
 -------------------
 
 * Allow user to specify different local backends.
+* Fix two-qubit unitary conversions.
```

Comment:
This line shouldn't be here (was a mistake, now removed on `develop`).

Reply:
My bad